specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          "ContainerAppsJobs"
+        ],
+        "summary": "Delete a Container Apps Job.",
+        "description": "Delete a Container Apps Job.",
+        "operationId": "ContainerAppsJob_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Container Apps Job deleted successfully."
+          },
+          "202": {
+            "description": "Delete operation is in progress."
+          },
+          "204": {
+            "description": "Container Apps Job does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "Update properties of a Container Apps Job",
+        "description": "Patches a Container Apps Job using JSON Merge Patch",
+        "operationId": "ContainerAppsJob_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerAppsJobEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties of a Container Apps Job that need to be updated",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsJob"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
No need of a 200 because there will never be a scenario we can patch a job in real time. Just 202 is enough.

Reply:
let's see what the response is for Container Apps and follow the same pattern

Reply:
@vinisoto, @trajkobal: following the same pattern as we have for Container Apps (https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json#L296)

Reply:
updated